include/config.hpp

```diff
@@ -13,6 +13,18 @@
 /*
  * Serial config
  */
+// uncomment the below line to use a seperate UART port for radio communication
+// this frees UART0 and the USB port for programming
+#define USE_UART2_FOR_RADIO  // UART2 is readily useable, UART1 requires more configuration as it shares pins with SPI
+#ifdef USE_UART2_FOR_RADIO
+    #include <HardwareSerial.h>
```

Comment:
Don't mix `#define`s and `#include`s in the code. Move the `#include` to the top

Reply:
On it. Thank you.